build.gradle

```diff
@@ -28,14 +28,30 @@ group= "vexatos.tgregworks" // http://maven.apache.org/guides/mini/guide-naming-
 archivesBaseName = "TGregworks"
 
 repositories {
-    /*maven {
-        name 'CB Maven FS'
-        url "http://chickenbones.net/maven/"
-    }
     maven {
-        name "DVS1 Maven FS'"
-        url 'http://dvs1.progwml6.com/files/maven'
-    }*/
+        name = 'Forge'
+        url = 'http://files.minecraftforge.net/maven'
+    }
+    
+    maven { // TConstruct, Mantle
+        name = 'DVS1 Maven FS'
+        url = 'http://dvs1.progwml6.com/files/maven'
+    }
+
+    ivy {
+        name = 'gtnh_download_source_stupid_underscore_typo'
+        artifactPattern("http://downloads.gtnewhorizons.com/Mods_for_Jenkins/[module]_[revision].[ext]")
+    }
+
+    ivy { // GregTech
+        name = 'gtnh_download_source'
+        artifactPattern("http://downloads.gtnewhorizons.com/Mods_for_Jenkins/[module]-[revision].[ext]")
```

Comment:
I would prefer to depend on official GregTech sources; modifying the build files should not be part of this PR.

Reply:
In fact, I cannot build with the original build.gradle (lots of dependencies cannot be found), so I use dependencies from GTNH's repo. Same reason for the change of GT version.
If you are sure you can build with the original build.gradle, I will reset this file.

Reply:
TGw should stay compatible, and therefore build, with the last official GregTech version, at least for now. Regardless, updating the build instructions has nothing to do with the localization options proposed in this PR.

Reply:
Ok, I will rollback these two files. But I hope you could update the unavailable dependencies, in a compatible way. A useless but compatible gradle file is unreasonable.